examples/using-wordpress/src/components/styled/index.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import styled, { injectGlobal, extend } from 'styled-components';
+import Link from 'gatsby-link';
+import * as PR from './propReceivers';
+import theme from './theme';
+import { fontFaceHelper } from '../../utils/fontFaceHelper';
+export {
+  Page,
+  Row,
+  Column
+} from './layout';
+
+
+const { ftz, color } = theme;
+
+
+
+/* Global Styles */
+injectGlobal`
+  ${fontFaceHelper('butler', 'butler_black-webfont')}
+  ${fontFaceHelper('butler', 'butler_black_stencil-webfont', 'normal')}
+  ${fontFaceHelper('butler', 'butler_regular-webfont')}
+  body {
+    font-family: 'butler', sans-serif;
+    margin: 0;
+    background-color: ${color.lightGray};
+  }
+`;
+
+
+/* Global Tags */
+export const H1 = styled.h1`
+  ${props => PR.ftzProps(props, ftz.h1)}
+  ${props => PR.colorProps(props)}
```

Comment:
For your consideration...

This could be DRYed up *a lot*. for example, something like this (untested) maybe?
```js
const styleHeader level => {
  const hx = `h${level}`;
  return styled[hx]`
    ${props => PR.ftzProps(props, ftz[hx])}
    ${props => PR.colorProps(props)}
    ${PR.marginProps}
  `;
};
```
then
```js
const [H1, H2, H3, H4, H5, H6]  = [1, 2, 3, 4, 5, 6].map(styleHeader);
```